config/testreport/css/default.css

```diff
@@ -1599,7 +1612,19 @@ tr:hover td.n60 , tr:hover td.n61 , tr:hover td.n62 , tr:hover td.n63 , tr:hover
+tr:hover td.a10, tr:hover td.a11, tr:hover td.a12, tr:hover td.a13, tr:hover td.a14, tr:hover td.a15, tr:hover td.a16, tr:hover td.a17, tr:hover td.a18, tr:hover td.a19 { background: linear-gradient(rgba(222,230,237,0.7), rgba(0,0,0,0), rgba(222,230,237,0.7)) !important; }
+tr:hover td.a20, tr:hover td.a21, tr:hover td.a22, tr:hover td.a23, tr:hover td.a24, tr:hover td.a25, tr:hover td.a26, tr:hover td.a27, tr:hover td.a28, tr:hover td.a29 { background: linear-gradient(rgba(206,217,228,0.7), rgba(0,0,0,0), rgba(206,217,228,0.7)) !important; }
+tr:hover td.a30, tr:hover td.a31, tr:hover td.a32, tr:hover td.a33, tr:hover td.a34, tr:hover td.a35, tr:hover td.a36, tr:hover td.a37, tr:hover td.a38, tr:hover td.a39 { background: linear-gradient(rgba(189,204,219,0.7), rgba(0,0,0,0), rgba(189,204,219,0.7)) !important; }
+tr:hover td.a40, tr:hover td.a41, tr:hover td.a42, tr:hover td.a43, tr:hover td.a44, tr:hover td.a45, tr:hover td.a46, tr:hover td.a47, tr:hover td.a48, tr:hover td.a49 { background: linear-gradient(rgba(173,191,210,0.7), rgba(0,0,0,0), rgba(173,191,210,0.7)) !important; }
+tr:hover td.a50, tr:hover td.a51, tr:hover td.a52, tr:hover td.a53, tr:hover td.a54, tr:hover td.a55, tr:hover td.a56, tr:hover td.a57, tr:hover td.a58, tr:hover td.a59 { background: linear-gradient(rgba(156,178,201,0.7), rgba(0,0,0,0), rgba(156,178,201,0.7)) !important; }
+tr:hover td.a60, tr:hover td.a61, tr:hover td.a62, tr:hover td.a63, tr:hover td.a64, tr:hover td.a65, tr:hover td.a66, tr:hover td.a67, tr:hover td.a68, tr:hover td.a69 { background: linear-gradient(rgba(140,165,192,0.7), rgba(0,0,0,0), rgba(140,165,192,0.7)) !important; }
+tr:hover td.a70, tr:hover td.a71, tr:hover td.a72, tr:hover td.a73, tr:hover td.a74, tr:hover td.a75, tr:hover td.a76, tr:hover td.a77, tr:hover td.a78, tr:hover td.a79 { background: linear-gradient(rgba(123,152,183,0.7), rgba(0,0,0,0), rgba(123,152,183,0.7)) !important; }
+tr:hover td.a80, tr:hover td.a81, tr:hover td.a82, tr:hover td.a83, tr:hover td.a84, tr:hover td.a85, tr:hover td.a86, tr:hover td.a87, tr:hover td.a88, tr:hover td.a89 { background: linear-gradient(rgba(107,139,174,0.7), rgba(0,0,0,0), rgba(107,139,174,0.7)) !important; }
+tr:hover td.a90, tr:hover td.a91, tr:hover td.a92, tr:hover td.a93, tr:hover td.a94, tr:hover td.a95, tr:hover td.a96, tr:hover td.a97, tr:hover td.a98, tr:hover td.a99 { background: linear-gradient(rgba(91,126,164,0.7), rgba(0,0,0,0), rgba(91,126,164,0.7)) !important; }
+tr:hover td.a100, tr:hover .infinity.neutral                                   																						     { background: linear-gradient(rgba(81,114,148,0.7), rgba(0,0,0,0), rgba(81,114,148,0.7)) !important; }                                                                                                                                                              { background: linear-gradient(rgb(200,120,120), rgba(0,0,0,0), rgb(200,120,120)) !important; }
```

Comment:
Looks like the CSS is somewhat damaged. The `{ background: linear-gradient(rgb(200,120,120), rgba(0,0,0,0), rgb(200,120,120)) !important; }` part at the end of the line belongs to the `tr:hover td.n100` line above.